src/app/login/models/feedback.model.ts

```diff
@@ -0,0 +1,17 @@
+export class Feedback {
+  public subject: string;
```

Comment:
In TS public is the default visibility modifier. Since we are not using it in the components I would aslo remove it here